src/mono/browser/runtime/startup.ts

```diff
@@ -534,17 +534,21 @@ async function maybeSaveInterpPgoTable() {
     await interp_pgo_save_data();
 }
 
-export function mono_wasm_load_runtime(unused?: string, debugLevel?: number): void {
+export function mono_wasm_load_runtime(config: MonoConfigInternal): void {
     mono_log_debug("mono_wasm_load_runtime");
     try {
         const mark = startMeasure();
+        let debugLevel = config.debugLevel;
         if (debugLevel == undefined) {
             debugLevel = 0;
             if (runtimeHelpers.config.debugLevel) {
                 debugLevel = 0 + debugLevel;
             }
         }
-        cwraps.mono_wasm_load_runtime(unused || "unused", debugLevel);
+        if (!loaderHelpers.isDebuggingSupported() || !config.resources!.pdb) {
+            debugLevel = 0;
+        }
+        cwraps.mono_wasm_load_runtime("unused", debugLevel);
```

Comment:
I don't know if `mono_wasm_load_runtime` is an API for somebody, but I guess we could drop the un-used param all the way to `driver.c`

Reply:
I thought it's in the unmanaged side for some reason, otherwise you would have removed it already. I'll try and we'll see 🤞